docs/.custom_wordlist.txt

```diff
@@ -45,3 +45,25 @@ VMs
 VPN
 websockets
 yaml
+STSStack
+sshuttle
+vpn
+behavior
+HH
+YYYY
+ss
+SQLAlchemy
+Caracal
+Xena
+fbcab
+ccb
+aac
+de
+acf
+ef
+ec
```

Comment:
Please remove these extra non-words now; the spellcheck shouldn't complain now that the hex identifier strings are in code blocks.

```suggestion
```